jobsdb/jobsdb_import.go

```diff
@@ -135,12 +135,15 @@ func (jd *HandleT) GetMaxIDForDs(ds dataSetT) int64 {
 	var maxID sql.NullInt64
 	sqlStatement := fmt.Sprintf(`SELECT MAX(job_id) FROM %s`, ds.JobTable)
 	row := jd.dbHandle.QueryRow(sqlStatement)
-	row.Scan(&maxID)
+	err := row.Scan(&maxID)
+	if err != nil {
```

Comment:
what if there is no row in the jobs table, should we really panic in such a scenario?

Reply:
Since it is an aggregation function, a single row should always be returns